.github/workflows/check-winlogbeat.yml

```
@@ -0,0 +1,28 @@
+name: check-winlogbeat
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/check-winlogbeat.yml'
+      - 'winlogbeat/**'
+      - 'x-pack/winlogbeat/**'
+
+env:
+  BEAT_MODULE: 'winlogbeat'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Fetch Go version from .go-version
+      run: echo "GO_VERSION=$(cat .go-version)" >> $GITHUB_ENV
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.GO_VERSION }}
+    - name: Run check/update
+      run: |
+        export PATH=$PATH:$(go env GOPATH)/bin
+        go install github.com/magefile/mage@latest
```

Removing the `@latest` will install the version used by the projects `go.mod` file. This would keep CI aligned to the version used by the project.

```suggestion
        go install github.com/magefile/mage
```